src/docker/tools/index.go

```diff
@@ -61,12 +61,9 @@ func GetExitCode(containerName string) (int, error) {
 }
 
 // GetRenderedVolumes returns the rendered paths to the given volumes
-func GetRenderedVolumes(volumes []string, appName string, role string, homeDir string) ([]string, error) {
-	dataPath := path.Join(homeDir, ".exosphere", appName, role, "data")
+func GetRenderedVolumes(volumes []string, appName string, role string) ([]string, error) {
+	dataPath := path.Join("${APP_PATH}", ".exosphere", "data", role)
 	renderedVolumes := []string{}
-	if err := util.MakeDirectory(dataPath); err != nil { //nolint gas
```

Comment:
We'll still have to create the dir at whatever `${APP_PATH}/.exosphere/data/role` compiles to right?

Reply:
True, I'll add that back